src/main/scala/com/gu/dynamodbswitches/Switches.scala

```diff
@@ -33,13 +34,14 @@ trait Switches extends Logging {
   }
 
   def updateDynamo(switches : List[Switch]) = {
-    val listWR = switches.map(switch =>
-        new WriteRequest(
-            new PutRequest(Map("name" -> switch.toStringAttribute(),
-                               "enabled" -> switch.asAttributeValue(switch.default)).asJava
-            )
-        )
-      ).asJava
+    val listWR = switches.map(switch => {
+      val putRequest: PutRequest = PutRequest.builder()
+        .item(Map("name" -> switch.toStringAttribute(),
+          "enabled" -> switch.asAttributeValue(switch.default)).asJava)
+        .build()
+      val writeRequest: WriteRequest = WriteRequest.builder().putRequest(putRequest).build()
+      writeRequest
+    }).asJava
     try {
       dynamoDbClient.batchWriteItem(
         BatchWriteItemRequest.builder()
```

Comment:
on line 48 (previously line 46), team city is showing the following error, which is a bit cryptic.
Any idea how to solve this?
```sh
[Error] /opt/teamcity/buildAgent/work/dfaeb46a61889340/src/main/scala/com/gu/dynamodbswitches/Switches.scala:46: type mismatch;
 found   : java.util.Map[String,java.util.List[Nothing]]
 required: java.util.Map[String, ? <: java.util.Collection[software.amazon.awssdk.services.dynamodb.model.WriteRequest]]
Note: java.util.List[Nothing] >: ? <: java.util.Collection[software.amazon.awssdk.services.dynamodb.model.WriteRequest], but Java-defined trait Map is invariant in type V.
You may wish to investigate a wildcard type such as `_ >: ? <: java.util.Collection[software.amazon.awssdk.services.dynamodb.model.WriteRequest]`. (SLS 3.2.10)
```